client/src/components/Common/Header/style.css

```diff
@@ -0,0 +1,57 @@
+.header-component {
+  background-color: #126fc9;
+  display: flex;
+  flex-direction: row-reverse;
+  justify-content: space-between;
+  align-items: center;
+  height: 3rem;
+}
+
+.flex-row {
+  display: flex !important;
+  flex-direction: row-reverse;
+  align-items: center;
+}
```

Comment:
try to follow bem naming convention while writing css styles. 
see more here 
http://getbem.com/naming/
```suggestion
.header__component {
  background-color: #126fc9;
  display: flex;
  flex-direction: row-reverse;
  justify-content: space-between;
  align-items: center;
  height: 3rem;
}

.flex__row {
  display: flex !important;
  flex-direction: row-reverse;
  align-items: center;
}
```
and don't forget to change them in the code